zh-Hans.lproj/Localizable.strings

```diff
@@ -15,21 +15,21 @@
 "Latest" = "最新版";
 "Specific" = "特定版本";
 
-"Please confirm that you have adequate free storage before proceeding:" = "请确认您有足够的存储空间再进行操作:";
-"Choose the backup you'd like to restore from:" = "请选择您要恢复的备份:";
-"Are you sure that you want to restore from %@?" = "您确定要从以下备份进行恢复吗 %@?";
+"Please confirm that you have adequate free storage before proceeding:" = "请确认您有足够的存储空间再进行操作：";
```

Comment:
Is there a reason for the extra spaces added after various punctuation?

Reply:
No, that's not a space. I will modify it to Chinese to use symbols“：”